internal/venice-common/src/main/java/com/linkedin/venice/writer/VeniceWriterOptions.java

```diff
@@ -55,18 +59,26 @@ public Time getTime() {
     return time;
   }
 
-  public Optional<Integer> getPartitionCount() {
+  public Integer getPartitionCount() {
     return partitionCount;
   }
 
   public boolean isChunkingEnabled() {
     return chunkingEnabled;
   }
 
+  public boolean isChunkingSet() {
```

Comment:
Why do we need this?

Reply:
It was intended to be used in `VeniceWriter` but it seems unnecessary. 

Deleted.